src/event_listener_tests/test_auth.py

```
@@ -0,0 +1,246 @@
+        auth1 = get_github_auth()
+        auth2 = get_github_auth()
+
+        assert auth1 is auth2
+
+    @pytest.mark.asyncio
+    async def test_cleanup_auth(self):
+        """Test auth cleanup."""
+        from event_listener.github_auth import _auth_instance, cleanup_auth
+
+        # Create instance
+        auth = get_github_auth()
```

Ruff (0.11.9)

Local variable `auth` is assigned to but never used

Remove assignment to unused variable `auth`

(F841)